apollo-portal/src/main/resources/static/views/component/release-modal.html

```diff
@@ -51,103 +51,133 @@ <h4 class="modal-title" ng-show="toReleaseNamespace.isBranch && toReleaseNamespa
+                                </td>
+                                <td width="25%" title="{{config.newValue}}">
+                                    <span ng-bind="config.newValue"></span>
+                                </td>
+                                <td width="10%" ng-bind="config.item.dataChangeLastModifiedBy">
+                                </td>
+                                <td width="10%"
+                                    ng-bind="config.item.dataChangeLastModifiedTime | date: 'yyyy-MM-dd HH:mm:ss'">
+                                </td>
+                                <td width="10%">
+                                    <span>
+                                        <button type="button" class="btn btn-success"
+                                                ng-click="compare(config,0)">{{'Component.Publish.Text.Compare' | translate
+                                        }}
+                                    </button>
+                                    <button type="button" style="margin-top: 20px"  class="btn btn-primary"
+                                            ng-click="compare(config,1)">{{'Component.Publish.Json.Compare' | translate
+                                        }}
+                                    </button>
+                                    </span>
+                                </td>
+                            </tr>
                             </tbody>
                         </table>
 
                         <!--branch gray release-->
                         <table class="table table-bordered table-striped text-center table-hover" ng-if="toReleaseNamespace.isPropertiesFormat &&
                                toReleaseNamespace.isBranch && !toReleaseNamespace.mergeAndPublish">
                             <thead>
-                                <tr>
-                                    <th>
-                                        {{'Component.Publish.Key' | translate }}
-                                    </th>
-                                    <th>
-                                        {{'Component.Publish.MasterValue' | translate }}
-                                    </th>
-                                    <th>
-                                        {{'Component.Publish.GrayPublishedValue' | translate }}
-                                    </th>
-                                    <th>
-                                        {{'Component.Publish.GrayNoPublishedValue' | translate }}
-                                    </th>
-                                    <th>
-                                        {{'Component.Publish.ModifyUser' | translate }}
-                                    </th>
-                                    <th>
-                                        {{'Component.Publish.ModifyTime' | translate }}
-                                    </th>
-                                </tr>
+                            <tr>
```

Comment:
please keep the origin format to avoid unnecessary changes

Reply:
ok,I will do it.Thank you